docs/guide/deploy.md

```diff
@@ -131,4 +131,50 @@ pages:
 
 3. Deploy to surge, by typing `surge docs/.vuepress/dist`.
 
-You can also deploy to a [custom domain](http://surge.sh/help/adding-a-custom-domain) by adding `surge docs/.vuepress/dist yourdomain.com`.
\ No newline at end of file
+You can also deploy to a [custom domain](http://surge.sh/help/adding-a-custom-domain) by adding `surge docs/.vuepress/dist yourdomain.com`.
```

Comment:
These two lines looks the same, I don't know why they are diffs？

Reply:
believe me I do not know either, just add a new line to start with the deploy in heroku
Any recommendation?